### ISSUE TITLE
function i18n_status_func not found

### ISSUE THREAD

**Issue by jorisroling** [francocatena/meteor-status#22]
I had to implement the following on the client to make it work.

```js
i18n_status_func = function(key,options) {
	return TAPi18n.__(key,options);
}
```

**Comment by sferoze** [comment]
I tried this but the connection bar does not show anymore

**Comment by jorisroling** [comment]
Care to elaborate?

**Comment by sferoze** [comment]
@jorisroling yah I am using this package and got the same error after upgrading to meteor 1.2

`function i18n_status_func not found`

I tried adding the code snippet you provided in my Meteor.startup function on the client

I ran the app and the connection banner would not show up even with Meteor.disconnect() or stopping the server

Should I put it outside Meteor.startup?

**Comment by jorisroling** [comment]
I've put it in **client/status.js**, and yes, outside **Meteor.startup**...

**Comment by sferoze** [comment]
@jorisroling Strange, I just tried it outside Meteor.startup and still the banner does not show. I still get the error: `function i18n_status_func not found` as well

**Comment by jorisroling** [comment]
You used my exact synatx?

**Comment by francocatena** [comment]
I remove all the _customization_ of tap:i18n, now I'm using it as plain as I can. Now it work well with Meteor 1.2 and this problem should be gone for good. The release with the fix is 1.4.0.

Cheers.
